### PR TITLE
New package: musikcube

### PR DESCRIPTION
**Pull request by the-eater**

### CODE REVIEW

File: srcpkgs/musikcube/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'musikcube'
+pkgname=musikcube
+version=0.60.1
+revision=1
+build_style=cmake
+configure_args="-DNO_NCURSESW=1"
+makedepends="boost-devel libogg-devel libmicrohttpd-devel ffmpeg-devel libressl-devel pulseaudio-devel lame-devel zlib-devel alsa-lib-devel libvorbis-devel libcurl-devel ncurses-devel libev-devel taglib-devel"
```
Comment (Johnnynator) [inline srcpkgs/musikcube/template]: Add a newline after 80 char.

Comment (the-eater) [reply srcpkgs/musikcube/template]: that's not in xlint! :(

File: srcpkgs/musikcube/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'musikcube'
+pkgname=musikcube
+version=0.60.1
+revision=1
+build_style=cmake
+configure_args="-DNO_NCURSESW=1"
+makedepends="boost-devel libogg-devel libmicrohttpd-devel ffmpeg-devel \
+	libressl-devel pulseaudio-devel lame-devel zlib-devel alsa-lib-devel \
```
Comment (maxice8) [inline srcpkgs/musikcube/template]: no need for the `\` and use whitespace not tabs to indent newlines in the same variable

File: srcpkgs/musikcube/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+checksum=7b6661673952a0a27310053ad19defea313a70f4c70aecc91e4a5d1a71bde13c
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+musikcube-devel_package() {
+	depends="musikcube_${version}_${revision}"
+	short_desc+=" - development files"
+
```
Comment (maxice8) [inline srcpkgs/musikcube/template]: unnecessary blank line

File: srcpkgs/musikcube/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+homepage="https://musikcube.com/"
+distfiles="https://github.com/clangen/musikcube/archive/${version}.tar.gz"
+checksum=7b6661673952a0a27310053ad19defea313a70f4c70aecc91e4a5d1a71bde13c
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+musikcube-devel_package() {
+	depends="musikcube_${version}_${revision}"
```
Comment (maxice8) [inline srcpkgs/musikcube/template]: first underscore (_) should be dash (-)

**Comment by maxice8** [comment]
Someone should tell upstream to install their ELFs in /usr/lib

File: srcpkgs/musikcube/template (hunk 4 above)
Comment (the-eater) [reply srcpkgs/musikcube/template]: Fixed, thanks :)

**Comment by the-eater** [comment]
@maxice8 I've included a patch to fix the install locations, I'll notify upstream too

**Comment by maxice8** [comment]
On Tue, Jan 08, 2019 at 09:37:26AM +0000, eater wrote:
> @maxice8 I've included a patch to fix the install locations, I'll notify upstream too
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/6731#issuecomment-452234010

Nice, thanks.

File: srcpkgs/musikcube/template [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ libressl-devel pulseaudio-devel lame-devel zlib-devel alsa-lib-devel
+ libvorbis-devel libcurl-devel ncurses-devel libev-devel taglib-devel"
+short_desc="Terminal-based music player in c++"
+maintainer="eater <=@eater.me>"
+license="BSD-3-Clause"
+homepage="https://musikcube.com/"
+distfiles="https://github.com/clangen/musikcube/archive/${version}.tar.gz"
+checksum=7b6661673952a0a27310053ad19defea313a70f4c70aecc91e4a5d1a71bde13c
+
+if [[ "${XBPS_TARGET_MACHINE}" == armv[56]* ]]; then
```
Comment (maxice8) [inline srcpkgs/musikcube/template]: it is more common to use a `case esac` structure for dealing with these

Comment (the-eater) [reply srcpkgs/musikcube/template]: fixed :)

**Pull request merged**